docs/porting/psa/crypto.md

```diff
@@ -15,11 +15,14 @@ Mbed OS currently provides two tests for Mbed Crypto:
 
 To run these tests, make sure make sure your target configuration is set:
 
-1.  `extra_labels` contains the label `PSA`. Please see an example using the [K64F](https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L1451) or [Future Sequana](https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L7694).
-1.  `MBEDTLS_PSA_CRYPTO_C` macro is enabled. Please see an example using the [K64F](https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L1454) or [Future Sequana](https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L7697).
-1. `MBEDTLS_ENTROPY_NV_SEED` macro is enabled in the SPE if the device does not have TRNG or if you want the entropy injection test. Please see an example using the [Future Sequana](https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L7673).
-1. `MBEDTLS_PLATFORM_NV_SEED_READ_MACRO` macro is set to `mbed_default_seed_read` in the SPE if the device does not have TRNG or if you want the entropy injection test. Please see an example using the [Future Sequana](https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L7674).
-1. `MBEDTLS_PLATFORM_NV_SEED_WRITE_MACRO` macro is set to `mbed_default_seed_write` in the SPE if the device does not have TRNG or if you want the entropy injection test. Please see an example using the [Future Sequana](https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L7674).
+1.  `extra_labels` contains the label `PSA`. Please see an example using the [K64F](https://github.com/ARMmbed/mbed-os/blob/mbed-os-5.13.0-rc2/targets/targets.json#L1485) or [LPC55S69](https://github.com/ARMmbed/mbed-os/blob/mbed-os-5.13.0-rc2/targets/targets.json#L2129).
```

Comment:
I know we've been trying to get away from board-specific examples. Do we need to include these?

Reply:
I got rid of most of them. We can get rid of any remaining, sure.